alacritty_terminal/src/tty/windows/winpty.rs

```diff
@@ -32,8 +31,9 @@ use crate::tty::windows::Pty;
 
 pub use winpty::Winpty as Agent;
 
-/// How long the winpty agent should wait for any RPC request
-/// This is a placeholder value until we see how often long responses happen
+/// How long the winpty agent should wait for any RPC request.
```

Comment:
```suggestion
/// How long the WinPTY agent should wait for any RPC request.
```

Some minor change for consistency might be worthwhile if we change things anyways.

Reply:
I feel like it should be `WinPty`, since we're using `Pty` for other things.